# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/imst.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- xc/programs/Xserver/hw/xfree86/drivers/imstt/imstt_reg.h.orig	2002-08-06 21:57:30.000000000 +0200
+++ xc/programs/Xserver/hw/xfree86/drivers/imstt/imstt_reg.h	2004-02-18 16:24:47.000000000 +0100
@@ -9,7 +9,7 @@
 
 static __inline__ void regw(unsigned long base_addr, unsigned long regindex, unsigned long regdata)
 {
-	asm volatile ("stwbrx %1,%2,%3; eieio"
+	__asm__ __volatile__ ("stwbrx %1,%2,%3; eieio"
 		      : "=m" (*(volatile unsigned *)(base_addr+regindex))
 		      : "r" (regdata), "b" (regindex), "r" (base_addr));
 }
@@ -19,7 +19,7 @@
 {
 	register unsigned long val;
 
-	asm volatile ("lwbrx %0,%1,%2; eieio"
+	__asm__ __volatile__ ("lwbrx %0,%1,%2; eieio"
 		      : "=r" (val)
 		      : "b" (regindex), "r" (base_addr),
 		      "m" (*(volatile unsigned *)(base_addr+regindex)));
